mmv1/products/siteverification/WebResource.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'WebResource'
+description: |
+  A web resource is a website or domain with verified ownership. Once your
+  ownership is verified you will be able to manage your website in the
+  [Google Search Console](https://www.google.com/webmasters/tools/).
+
+  ~> **Note:** The verification data (DNS `TXT` record, HTML file, `meta` tag, etc.)
+  must already exist before the web resource is created, and must be deleted before
+  the web resource is destroyed. The Google Site Verification API checks that the
+  verification data exists at creation time and does not exist at destruction time
+  and will fail if the required condition is not met.
+base_url: webResource
+self_link: webResource/{{id}}
+create_url: webResource?verificationMethod={{verification_method}}
+delete_url: webResource/{{id}}
+id_format: '{{id}}'
```

Comment:
Should this be `webResource/{{id}}`? Usually we keep this in a selfLink-ish form so we can use .id from other resources. In this case that would probably never happen, but it would keep the ID formats standardized

Reply:
In this case, changing `id_format` results in a post-create read to `/siteVerification/v1/webResource/webResource/{{id}}`.

FWIW I have a follow-up PR to add a fine grained resource to this resource, and the test resource looks like:

```
resource "google_site_verification_owner" "example" {
  web_resource_id = google_site_verification_web_resource.example.id
  email           = "%{owner}"
}
```

It resolves OK 🤷